Branch-SDK/Branch-SDK/BranchView.h

```diff
@@ -0,0 +1,53 @@
+//
+//  BranchView.h
+//  Branch-TestBed
+//
+//  Created by Sojan P.R. on 3/4/16.
+//  Copyright © 2016 Branch Metrics. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+
+@interface BranchView : NSObject
+//-------- properties-------------------//
+/**
+ Unique ID for this Branch view
+ */
+@property (strong, nonatomic) NSString *branchViewID;
```

Comment:
I guess I'm unclear as to what the view branchView is, so ID is unclear, action is unclear, number of times it can be used is unclear (`used` for what?). If I could see a spec (share via Slack) or documentation, that would go a long ways.